Apromore-Custom-Plugins/Apromore-Editor/bpmn20xmlbasic/src/test/java/com/processconfiguration/common/Constants.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * #%L
+ * This file is part of "Apromore Enterprise Edition".
+ * %%
+ * Copyright (C) 2019 - 2021 Apromore Pty Ltd. All Rights Reserved.
+ * %%
+ * NOTICE:  All information contained herein is, and remains the
+ * property of Apromore Pty Ltd and its suppliers, if any.
+ * The intellectual and technical concepts contained herein are
+ * proprietary to Apromore Pty Ltd and its suppliers and may
+ * be covered by U.S. and Foreign Patents, patents in process,
+ * and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this
+ * material is strictly forbidden unless prior written permission
+ * is obtained from Apromore Pty Ltd.
+ * #L%
+ */
```

Comment:
This should be the licence header for ApromoreCore. You can run ./gradlew lF in the ApromoreCore repo to add the licence headers. Running the command in the ApromoreEE repo will add the ApromoreEE licence header to files missing the license.

Reply:
Good pick! Thanks Jane